src/clientsettingsdlg.cpp

```diff
@@ -652,7 +652,7 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient*         pNCliP,
     QObject::connect ( tabSettings, &QTabWidget::currentChanged,
         this, &CClientSettingsDlg::OnTabChanged );
 
-    tabSettings->setCurrentIndex ( iTabIdx );
+    tabSettings->setCurrentIndex ( pSettings->iSettingsTab ); //pSettings->iSettingsTab
```

Comment:
I guess the comment was whilst copy/pasting?  It can go, right?

Reply:
Oh, yes, of course